contracts/NFTs/VehicleId.sol

```diff
@@ -80,6 +80,35 @@ contract VehicleId is Initializable, MultiPrivilege {
         trustedForwarders[addr] = trusted;
     }
 
+    /**
+     * @notice Gets the data URI associated to a token
+     * @param tokenId Token Id to be checked
+     * @return data Data URI
+     */
+    function getData(uint256 tokenId)
```

Comment:
The name seems odd to me. You're not getting the data!

`dataURI`? My guess is you're gonna want `dataUri` but note that it's `tokenURI` in the spec.